main.py

```diff
@@ -4,58 +4,115 @@
           "result_limit": <integer limit number for max results in the data source query>,
-          "timerange": <integer time window in minutes used in the data source query when START STOP qualifiers are absent>
+          "timerange": <integer time range for LAST x MINUTES used in the data source query when START STOP qualifiers are absent>
        }"
+    In the case of transmission, stix-shifter connects to a datasource to execute queries, status updates, and result retrieval.
+    Arguments will take the form of...
+    "transmit" <module> '{"host": <host IP>, "port": <port>, "cert": <certificate>}', '{"auth": <authentication>}',
+        <
+            query <query string>,
+            status <search id>,
+            results <search id> <offset> <length>,
+            ping,
+            is_async
+        >
     """
 
     # process arguments
-    parser = argparse.ArgumentParser(description='stix_shifter')
-    subparsers = parser.add_subparsers(dest='command')
+    parent_parser = argparse.ArgumentParser(description='stix_shifter')
+    parent_subparsers = parent_parser.add_subparsers(dest='command')
 
     # translate parser
-    translate_parser = subparsers.add_parser(
-        'translate', help='Translate a query or result set using a specific translation module')
+    translate_parser = parent_subparsers.add_parser(
+        TRANSLATE, help='Translate a query or result set using a specific translation module')
+
     # positional arguments
     translate_parser.add_argument(
-        'module', choices=stix_shifter.MODULES, help='what translation module to use')
+        'module', choices=stix_shifter.TRANSLATION_MODULES, help='what translation module to use')
     translate_parser.add_argument('translate_type', choices=[
         stix_shifter.RESULTS, stix_shifter.QUERY], help='what translation action to perform')
     translate_parser.add_argument(
         'data_source', help='STIX identity object representing a datasource')
     translate_parser.add_argument(
         'data', type=str, help='the data to be translated')
     translate_parser.add_argument('options', nargs='?', help='options that can be passed in')
+
     # optional arguments
     translate_parser.add_argument('-x', '--stix-validator', action='store_true',
                                   help='run stix2 validator against the converted results')
     translate_parser.add_argument('-m', '--data-mapper',
                                   help='module to use for the data mapper')
 
-    args = parser.parse_args()
+    # transmit parser
+    transmit_parser = parent_subparsers.add_parser(
+        TRANSMIT, help='Connect to a datasource and exectue a query...')
+
+    # positional arguments
+    transmit_parser.add_argument(
+        'module', choices=stix_shifter.TRANSMISSION_MODULES,
+        help='choose which connection module to use'
+    )
+    transmit_parser.add_argument(
+        'connection',
+        type=str,
+        help='Data source Host'
```

Comment:
could you please fix this help message?
I know it was like this in transmission, my miss